README.md

```diff
@@ -35,7 +35,7 @@ https://github.com/orangehrm/orangehrm/wiki/Resources
 
 
 ## Demo
-Live demo is available at : http://opensource.demo.orangehrmlive.com
+Live demo is available at : http://opensource.demo.orangehrmlive.com/index.php

```

Comment:
Why it's needed to redirect to index.php? it's should be part of apache/nginx server settings